package/src/common/Input/InputButton/InputButton.module.css

```diff
@@ -0,0 +1,119 @@
+/*------------------inputs--------------------*/
+/*container label+type*/
+.wrapper {
+    display: flex;
+    flex-direction: column;
+    font-size: 1rem;
+    min-width: 6rem;
+}
+
+.wrapper, 
+.wrapper * {
+    box-sizing: border-box;
+}
+
+.wrapper > label {
+    display: flex;
+    flex-direction: row;
+    align-items: baseline;
+    font-weight: 500;
```

Comment:
I see a lot of fonts sizes and styles being hardcoded for each class. Is there any way of having them separately defined as titles, headings, captions, etc.? It would help in the future for maintaining visual consistency throughout the app when adding new features in the future